.github/workflows/build.yml

```diff
@@ -24,8 +24,8 @@ env:
   DVC_TEST: "true"
   FORCE_COLOR: "1"
   DATASET: ${{ github.event.inputs.dataset || ( github.event_name == 'schedule' && 'mnist' || 'small' ) }}
-  DVC_REVS: main,2.45.0,2.41.1,2.40.0,2.39.0,2.18.1,2.11.0
-  CLOUD_REVS: main,2.41.1,2.40.0,2.39.0,2.11.0
+  DVC_REVS: main,3.0.0,2.58.2,2.40.0,2.11.0
```

Comment:
Should we do something between main and plain `3.0`? There were a lot of things going on in the minor versions

Reply:
True, but do we care about any of those version specifically?

Reply:
Hard to tell. I guess we care before and after using index for core operations but not sure to which minor version(s) that maps

Reply:
Once we have 3.0.0 and main, we can start playing with adding other versions where we see regressions to bisect the problem.

Reply:
sure thing, not a blocker